zebra/debug.c

```diff
@@ -476,7 +460,8 @@ DEFUN (no_debug_zebra_kernel,
        "Zebra configuration\n"
        "Debug option set for zebra between kernel interface\n")
 {
-	zebra_debug_kernel = 0;
+	UNSET_FLAG(zebra_debug_kernel, ZEBRA_DEBUG_KERNEL);
+
```

Comment:
hmm - since "kernel" is the parent of "msgdump", I thought this one was ok as it was. it seemed like "foo" and "foo detail" - if you disable "foo", you implicitly disable "foo detail" too?

Reply:
I don't agree that this is `foo detail` but I can see arguments for either way.